walk/config.go

```diff
@@ -78,14 +75,18 @@ type Configurer struct {
 	// Excludes and BUILD filenames specified on the command line.
 	// May be extending with BUILD directives.
 	cliExcludes       []string
+	cliFollow         []string
 	cliBuildFileNames string
 
 	// Alternate BUILD read/write directories
 	readBuildFilesDir, writeBuildFilesDir string
 }
 
 func (wc *Configurer) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Config) {
+	c.Exts[walkConfigurerName] = wc
+
 	fs.Var(&gzflag.MultiFlag{Values: &wc.cliExcludes}, "exclude", "pattern that should be ignored (may be repeated)")
+	fs.Var(&gzflag.MultiFlag{Values: &wc.cliFollow}, "follow", "pattern that should be followed (may be repeated)")
```

Comment:
Is there a reason this needs to be added as a command-line flag? This PR seems like it could be done by refactoring only without changing the CLI. If this stays in, then the CLI documentation in README.rst needs to be updated, too.

(If we didn't mind the incompatible change, I wish we could remove nearly all of the CLI flags. Directives are a more powerful, consistent mechanism. And it's better not to have two ways to do the same thing.)

Reply:
Yeah I want to drop this, it was needed to make a test pass for some reason. I'll have to look again and figure out another way to do it...